packages/transactions/src/OutboundShipment/ListView/ListView.tsx

```diff
@@ -117,8 +120,31 @@ export const OutboundShipmentListViewComponent: FC = () => {
     { onChangeSortBy, sortBy }
   );
 
+  const [open, setOpen] = useState(false);
+
   return (
     <>
+      <CustomerSearch
+        open={open}
+        onClose={() => setOpen(false)}
+        onChange={async name => {
+          setOpen(false);
+
+          const createInvoice = async () => {
+            const invoice = {
+              id: String(Math.ceil(Math.random() * 1000000)),
+              nameId: name?.id,
+            };
+
+            const result = await onCreate(invoice);
+
+            invalidate();
+            navigate(`/customers/customer-invoice/${result.invoiceNumber}`);
+          };
+
+          createInvoice();
+        }}
+      />
```

Comment:
With throttling it feels as if there needs to be something to give feedback. Maybe a blocking modal (which we might need for finalising, anyway). Thought to do it later, but something to consider?